test/unittests/test_utils.py

```diff
@@ -12,6 +12,58 @@
 from crmsh import tmpfiles
 
 
+def test_check_ssh_passwd_need():
+    with mock.patch('crmsh.utils.get_stdout_stderr') as mock_get_stdout_stderr:
+        mock_get_stdout_stderr.side_effect = [(0, None, None), (1, None, None)]
+        assert utils.check_ssh_passwd_need(["node1", "node2"]) == True
```

Comment:
@liangxin1300 we should also assert a False return case too.

Reply:
Yes, I defined this `def test_check_ssh_passwd_need_Flase():`